README.md

```diff
@@ -21,8 +21,23 @@ If you find a vulnerability that may affect live deployments -- for example, by
 If the issue is a protocol weakness that cannot be immediately exploited or something not yet deployed, just discuss it openly.
 
 ## [Project directory](project-directory.md)
-Lost? Searching for something? Need to submit an issue? Want to contribute? Take a peek at [project-directory.md](project-directory.md) which maps out all the repositories for the ipfs project on Github.
 
+The IPFS community has a lot of repositories:
+
+- [ipfs/ipfs](https://github.com/ipfs/ipfs) (this repo) is _the_ repo for our updates and news.
+- [ipfs/go-ipfs](https://github.com/ipfs/go-ipfs) is the main, reference implementation.
+- [ipfs/specs](https://github.com/ipfs/specs) contains the technical specs for IPFS.
+- [ipfs/faq](https://github.com/ipfs/faq) is for questions you might have about IPFS.
+- [ipfs/support](https://github.com/ipfs/support) is for issues with getting IPFS up and running.
+- [ipfs/community](https://github.com/ipfs/community) is for meta-questions on how we manage our community in general.
```

Comment:
Maybe we should have a link to the full directory here (redundancy is ok), and on website